lib/ArcadeFrame.dart

```diff
@@ -69,18 +71,25 @@ class _HomeScreenState extends State<_HomeScreen> {
   }
 
   loadFromUri(Uri uri) async {
+    print("loadFromUri");
     if (uri == null) return;
     var segs = uri.pathSegments;
     if (segs.length != 2 && segs[0] != "app") return;
-
+    print(segs[1]);
     Game game = await bloc.queryGame(segs[1]);
-    if (game == null) return;
-
+    if (game == null) {
+      return;
+    }
     bloc.viewGame(game, context, "QR");
   }
 
+
   @override
   Widget build(BuildContext context) {
+    var devicePhysicalPixelWidth = MediaQuery.of(context).size.width * MediaQuery.of(context).devicePixelRatio;
+    var headerPadding = (devicePhysicalPixelWidth > 1000) ? MediaQuery.of(context).size.width * 0.15 : 64;
```

Comment:
I'd move some of these magic numbers to constants.dart with nice variable names. Linda hard for me to fully image this